### PR TITLE
updating build timing types to fall in line with the ones in origin

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

**Comment by coreydaley** [comment]
@bparees ptal

File: pkg/util/status/build_status_test.go [hunk 1]
```diff
@@ -61,14 +61,14 @@ func TestUpdateStageDuration(t *testing.T) {
...
+	duration := buildInfo.Stages[0].DurationMilliseconds
 
 	addDuration, _ := time.ParseDuration("5m")
 
 	buildInfo.Stages = api.RecordStageAndStepInfo(buildInfo.Stages, api.StagePullImages, api.StepPullBuilderImage, time.Now(), time.Now().Add(addDuration))
 
-	if !(buildInfo.Stages[0].Duration > duration) {
-		t.Errorf("Stage Duration was not updated, expected %#v, got %#v", buildInfo.Stages[0].StartTime.Add(duration), buildInfo.Stages[0].Duration)
+	if !(buildInfo.Stages[0].DurationMilliseconds > duration) {
+		t.Errorf("Stage Duration was not updated, expected %#v, got %#v", buildInfo.Stages[0].StartTime.Add(time.Duration(duration)), buildInfo.Stages[0].DurationMilliseconds)
```
Comment (bparees) [inline pkg/util/status/build_status_test.go]: your duration variable here is a millisecond value, converting it to a time.Duration doesn't make sense unless you first scale it back to nanoseconds.

Comment (coreydaley) [inline pkg/util/status/build_status_test.go]: fixed

**Comment by coreydaley** [comment]
[test]

File: pkg/util/status/build_status_test.go [hunk 2]
```diff
@@ -61,14 +61,14 @@ func TestUpdateStageDuration(t *testing.T) {
...
+	duration := buildInfo.Stages[0].DurationMilliseconds
 
 	addDuration, _ := time.ParseDuration("5m")
 
 	buildInfo.Stages = api.RecordStageAndStepInfo(buildInfo.Stages, api.StagePullImages, api.StepPullBuilderImage, time.Now(), time.Now().Add(addDuration))
 
-	if !(buildInfo.Stages[0].Duration > duration) {
-		t.Errorf("Stage Duration was not updated, expected %#v, got %#v", buildInfo.Stages[0].StartTime.Add(duration), buildInfo.Stages[0].Duration)
+	if !(buildInfo.Stages[0].DurationMilliseconds > duration) {
+		t.Errorf("Stage Duration was not updated, expected %#v, got %#v", buildInfo.Stages[0].DurationMilliseconds+duration, buildInfo.Stages[0].DurationMilliseconds)
```
Comment (bparees) [inline pkg/util/status/build_status_test.go]: this error message doesn't make sense.

what you expected is that stage duration =~ duration+addDuration

Comment (coreydaley) [inline pkg/util/status/build_status_test.go]: Fixed, got in a hurry, should have just waited until later.

**Comment by coreydaley** [comment]
@bparees fixed, ptal

**Comment by bparees** [comment]
[merge]

**Pull request merged**